src/scss/utilities/_display.scss

```diff
@@ -1,7 +1,7 @@
-@include bp-suffix('u-d-no', $create-between-breakpoints: true) {
+@include bp-suffix('u-d-no', $createBetweenBreakpoints: true) {
```

Comment:
Changing the variables from hyphenated to pascal case will annoy the sass linter / codacy